core/utils.js

```diff
@@ -57,13 +57,6 @@ Blockly.utils.removeAttribute = function(element, attributeName) {
 };
 
 /**
- * Cached value for whether 3D is supported
- * @type {boolean}
```

Comment:
Interesting.  It looks like that's true in mainline blockly as well.  Yes, remove from both places and if we need to put it back we can d that later.

@picklesrus to get your eyes on this, since it was your code originally.